src/contracts/interfaces/IAtlas.sol

```diff
@@ -4,10 +4,22 @@ pragma solidity 0.8.22;
 import "../types/SolverCallTypes.sol";
 import "../types/UserCallTypes.sol";
 import "../types/DAppApprovalTypes.sol";
+import "../types/LockTypes.sol";
 
 // TODO add all Atlas functions here
 
 interface IAtlas {
+    function solverCall(
```

Comment:
This feels odd to put in the interface as its never intended to be called by third parties, only by Atlas itself in a try-catch. Think we should either add a note in the interface or find a way to not include it here

Reply:
Removed it here https://github.com/FastLane-Labs/atlas/pull/227/commits/a8e7096a079db4ddf74953b6a2f5fffb935f7818 and using this.solverCall instead